README.rst

```diff
@@ -3,8 +3,8 @@ Name My Sprint
 ==============
 
 This project will generate the name of your team's next sprint by
-combining a random adjective and a random name (chosen from a list of pro
-cyclists).
+combining a random adjective and a random name. There is a support
+for several name packs - for example cyclists, star wars characters, ...
```

Comment:
Capitalize Star Wars

Reply:
Make it say "There is currently support for several name packs (e.g. cyclists, Star Wars characters) and pull requests for new ones are encouraged."